orchestra/static/orchestra/common/js/orchestra.services.es6.js

```diff
@@ -64,14 +64,14 @@ export function orchestraService () {
 
```

Comment:
the only thing that changed in `style.css` above is the font stack moving over to the system fonts. the diff is large because of whitespace cleanup.